src/main/java/seedu/address/logic/commands/alias/AliasCommand.java

```diff
@@ -14,31 +14,40 @@
  */
 public class AliasCommand extends Command {
     public static final String COMMAND_WORD = "alias";
+
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sets a shortcut command for a longer command.\n"
             + "Parameters: a/ALIAS cmd/COMMAND\n"
             + "Example: " + COMMAND_WORD + " a/rl cmd/rlist";
 
     public static final String MESSAGE_SUCCESS = "Alias created: %1$s";
+
     public static final String MESSAGE_RESERVED_KEYWORD =
             "%1$s is a reserved keyword and cannot be used as an alias";
+
     public static final String MESSAGE_RECURSIVE =
             "Recursive alias is not allowed";
 
     private final Alias alias;
 
     /**
      * Creates an AliasCommand object.
+     * All field must not be null.
```

Comment:
Should `field` be `fields`?

Reply:
There is only one field so `field` is chosen

Reply:
Oh, then maybe change it to `The field must not be null.`